dotty-docker/Dockerfile

```diff
@@ -25,9 +25,11 @@ WORKDIR /var/cache/drone
 # Add ivy2 cache
 RUN git clone -b master --recursive https://github.com/lampepfl/dotty.git && \
     (cd dotty && \
-     sbt ";test:update ;dotty-bootstrapped/test:update ;sbt-dotty/scripted:update" && \
+     sbt ";test:update ;dotty-bootstrapped/test:update ;sbt-dotty/scripted:update ;community-build/test" && \
```

Comment:
This is going to be slow (and will get even slower if we start running tests in the community build), maybe instead customize the `update` key in `community-build` to run update on all the projects in the build ?

Reply:
I don't think there is a reliably way to update the subset of projects we actually test. Doing `sbt update` will download more things that we actually need (Scala.js stuff for example) so I'm not sure it would be a win...

Reply:
In https://github.com/lampepfl/dotty/blob/master/community-build/test/scala/dotty/communitybuild/CommunityBuildTest.scala for each project we do `x/compile` or `x/y:compile`, each of those depends on the corresponding `x/update` or `x/y:update` tasks, so that's what we should run I think.

Reply:
Doesn't work for the `dottyCompile` ones which are custom tasks

Reply:
Should be replaced by non-custom tasks :)